### PR TITLE
Mar 4645 inclusion of additional price statement

### PR DESCRIPTION
**Pull request by Revit**

### CODE REVIEW

**Review by Reshma-M-A — APPROVED** [review]

File: apps/marketplace/components/BuyerATM/BuyerATMIntroductionStage.js [hunk 1]
```diff
@@ -58,6 +58,7 @@ const BuyerATMIntroductionStage = props => (
           </li>
         </ul>
       </li>
+      <li>You should consider approaching multiple Sellers wherever possible.</li>
```
Comment (tobiasbrown) [inline apps/marketplace/components/BuyerATM/BuyerATMIntroductionStage.js:61]:
```suggestion
      <li>You should consider approaching multiple sellers wherever possible.</li>
```

File: apps/marketplace/components/BuyerRFX/BuyerRFXIntroductionStage.js [hunk 2]
```diff
@@ -50,6 +50,7 @@ const BuyerRFXIntroductionStage = props => (
           </li>
         </ul>
       </li>
+      <li>You should consider approaching multiple Sellers wherever possible.</li>
```
Comment (tobiasbrown) [inline apps/marketplace/components/BuyerRFX/BuyerRFXIntroductionStage.js:53]:
```suggestion
      <li>You should consider approaching multiple sellers wherever possible.</li>
```

File: apps/marketplace/components/BuyerSpecialist/BuyerSpecialistIntroductionStage.js [hunk 3]
```diff
@@ -51,6 +51,7 @@ const BuyerSpecialistIntroductionStage = props => (
           </li>
         </ul>
       </li>
+      <li>You should consider approaching multiple Sellers wherever possible.</li>
```
Comment (tobiasbrown) [inline apps/marketplace/components/BuyerSpecialist/BuyerSpecialistIntroductionStage.js:54]:
```suggestion
      <li>You should consider approaching multiple sellers wherever possible.</li>
```

File: apps/marketplace/components/BuyerTraining/BuyerTrainingIntroductionStage.js [hunk 4]
```diff
@@ -76,6 +76,7 @@ const BuyerTrainingIntroductionStage = props => (
           </li>
         </ul>
       </li>
+      <li>You should consider approaching multiple Sellers wherever possible.</li>
```
Comment (tobiasbrown) [inline apps/marketplace/components/BuyerTraining/BuyerTrainingIntroductionStage.js:79]:
```suggestion
      <li>You should consider approaching multiple sellers wherever possible.</li>
```

Comment (Revit) [reply apps/marketplace/components/BuyerTraining/BuyerTrainingIntroductionStage.js]: done

Comment (Revit) [reply apps/marketplace/components/BuyerTraining/BuyerTrainingIntroductionStage.js]: done

Comment (Revit) [reply apps/marketplace/components/BuyerTraining/BuyerTrainingIntroductionStage.js]: done

**Review by tobiasbrown — APPROVED** [review]

**Pull request merged**